pkg/collector/corechecks/containers/docker/adapter.go

```diff
@@ -38,8 +38,9 @@ var metricsNameMapping = map[string]string{
 	"container.pid.thread_count":      "docker.thread.count",
 	"container.pid.thread_limit":      "docker.thread.limit",
 	"container.pid.open_files":        "docker.container.open_fds",
-	"container.net.sent":              "",                      // Removed, handled by custom network extension
-	"container.net.rcvd":              "",                      // Removed, handled by custom network extension
+	"container.net.sent":              "", // Removed, handled by custom network extension
+	"container.net.rcvd":              "", // Removed, handled by custom network extension
+	"container.restarts":              "docker.restarts",
```

Comment:
We don't have metrics to existing containerd and docker checks. Only the new container checks gets new metrics.